test/BaselineOutput/Common/EntryPoints/core_manifest.json

```diff
@@ -9096,18 +9057,6 @@
           "IsNullable": false,
           "Default": false
         },
-        {
-          "Name": "MaxTreesAfterCompression",
-          "Type": "Int",
```

Comment:
why this got deleted?

Reply:
Not used at all.

---
In reply to: [262674075](https://github.com/dotnet/machinelearning/pull/2753#discussion_r262674075) [](ancestors = 262674075)